examples/global_concurrency_control.rb

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env ruby
+
+example_description = <<DESC
+  Concurrency Control Example
+  ===================
+
+  This example shows, how to ConcurrencyControl middleware
```

Comment:
Make sure the remove the middleware mentions, as it seems we're not going the middleware way anymore.